ReSharper.FSharp/src/FSharp.Psi.Daemon/src/Stages/FcsErrorsStageProcessBase.fs

```diff
@@ -470,7 +471,10 @@ type FcsErrorsStageProcessBase(fsFile, daemonProcess) =
 
             let expr = fsFile.GetNode<IDoLikeStatement>(range)
             if isNotNull expr then NamespaceCannotContainExpressionsError(expr) :> _ else null
-
+        
+        | UnionCaseDoesNotTakeArguments ->
```

Comment:
Please insert it next to `InstanceMemberRequiresTarget` and `UnionCaseExpectsTupledArgument` to keep the branches ordered by the error numbers.

Reply:
Updated